.cargo/config.toml

```diff
@@ -1,8 +1,11 @@
 [target.aarch64-unknown-linux-musl]
-rustflags = [ "-C", "target-feature=-crt-static" ]
+rustflags = ["-C", "target-feature=-crt-static", "--cfg", "tokio_unstable"]
 
 [target.x86_64-unknown-linux-musl]
-rustflags = [ "-C", "target-feature=-crt-static" ]
+rustflags = ["-C", "target-feature=-crt-static", "--cfg", "tokio_unstable"]
 
 [target.aarch64-unknown-linux-gnu]
 linker = "aarch64-linux-gnu-gcc"
+
+[build]
+rustflags = ["--cfg", "tokio_unstable"]
```

Comment:
We can't use console otherwise. We are building Drive in docker image with release profile and then we want to be able to use tokio console with this images on testnet or ozuo

Reply:
IMO this should be set in CI/CD, based on whether we do "real" release or just a dev snapshot

Reply:
We decided to keep console around for now and leave todo for production.